dev-docs/modules/consentManagement.md

```diff
@@ -188,6 +188,63 @@ $(function(){
 });
 </script>
 
+## Publishers not using an IAB-Compliant CMP
```

Comment:
I'd probably label this as a "non standard" or custom CMP - as it should still be IAB-compliant